### ISSUE TITLE
Hearthmod wont start due to couchbase/python module not installing properly

### ISSUE THREAD

**Issue by CrescentByte** [hearthmod/hearthmod#3]
Hi @farb3yonddriv3n

I watched your video guide and followed it 100%, however despite this my installation was not successful like in the video. basically what happened was when I tried to start hearthmod I ended up with:
``` 
Starting hearthmod
valgrind: ./hm_gameserver/hm_gameserver: No such file or directory
./hm_lobbyserver/hm_lobbyserver: error while loading shared libraries: libcouchbase.so.2: cannot open shared object file: No such file or directory
{core} Note: no DH parameters found in ./hm_stud/cert/test.com.pem
{core} Daemonized as pid 37659.
spawn-fcgi: child exited with: 1
```

I saved a copy of the installation log here: https://www.dropbox.com/s/jqn4p8gv0a7v1mv/hearthmod%20aug%202019.txt?dl=0

How do I correct this so that couchbase installs properly and allows me to run hearthmod?

Any assistance would be greatly appreciated.

**Comment by CrescentByte** [comment]
@farb3yonddriv3n just a quick update. 

I managed to get all the couchbase stuff working by making a fresh installation of the OS and installing all of the latest couchbase packages so I can now import the couchbase module (and all sub modules like .bucket) into python without any issues. My only question now is how do I go about editing your install script to skip your specific couchbase packages and use my pre-installed ones without distrupting any of the other package installation processes?

**Comment by CrescentByte** [comment]
@farb3yonddriv3n I have finally managed to run the script successfully by manually adding the hbs couchbase bucket and altering the main.py script to authenticate the connection to the bucket properly(tested in python before running the script). despite no serious error messages after running the script, the nginx webserver still wont connect.
here is a copy of my latest run of the script + my nginx error log:
https://www.dropbox.com/s/6foe57f0ssgu0rj/New%20log.txt?dl=0
https://www.dropbox.com/s/lhm4l9h8lp0rn0g/nginx%20log.txt

**Comment by CrescentByte** [comment]
Issue has been fixed by using ubuntu 18.04 and running the command 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib' after running the install script, then running the install script again.
